.gitignore

```diff
@@ -1,2 +1,4 @@
-/target
 /Cargo.lock
+/num_macros/Cargo.lock
+/num_macros/target
+/target
```

Comment:
I think it's also fine to just remove the leading `/` from the first two entries